### PR TITLE
Feature/#18

### LINKED ISSUE #18: Criar tabela Points no banco de dados
Criar no banco de dados a tabela points, para persistências dos locais cadastrados, com os seguintes campos
 - id
 - name
 - email
 - whatsapp
 - latitude
 - longitude
 - number
 - city
 - uf 
 - image

### PR DESCRIPTION
**Pull request by victor-luiz**
Executar pull request apenas quando for feito o pull request da tarefa #1

### CODE REVIEW

File: server/prisma/migrations/20210813210802_create_point/migration.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-- CreateTable
+CREATE TABLE "point" (
+    "id" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
```
Comment (Gustavo-franca) [inline server/prisma/migrations/20210813210802_create_point/migration.sql:4]: why not use VARCHAR , TEXT is too large for this case

**Review by Gustavo-franca — COMMENTED** [review]
I leave some Suggestions. 

I recommend rebase the last commits with develop branch.

File: server/prisma/migrations/20210813210802_create_point/migration.sql (hunk 1 above)
Comment (victor-luiz) [reply server/prisma/migrations/20210813210802_create_point/migration.sql:4]: o sistema do prisma cria as migrations sozinho

Comment (Gustavo-franca) [reply server/prisma/migrations/20210813210802_create_point/migration.sql:4]: tranquilo, só resolve o conflito e faz o merge

**Review by Gustavo-franca — APPROVED** [review]

**Pull request merged**